example_dags/example_load_file.py

```diff
@@ -151,4 +151,28 @@
     )
     # [END load_file_example_12]
 
+    # [START load_file_example_13]
```

Comment:
May be we should add one example of loading file from local as well as this DAG would test that scenario